src/components/comic/cards/DefaultCard.tsx

```diff
@@ -46,15 +47,9 @@ export const DefaultComicCard: React.FC<Props> = ({ comic, className }) => {
         <CopiesCount count={comic.stats?.issuesCount} withLabel />
       </div>
       <div className='flex flex-col px-2 pb-1 max-sm:gap-1'>
-        <Text
-          title={comic.title}
-          as='p'
-          styleVariant='body-normal'
-          fontWeight='bold'
-          className='line-clamp-1 overflow-ellipsis max-sm:text-sm'
-        >
+        <TextWithOverflow as='p' styleVariant='body-normal' fontWeight='bold' className='max-sm:text-sm'>
```

Comment:
This component is causing some issues on Comic Card. Check your version and https://www.dreader.app/

Reply:
I've added w-full missing tag in DefaultComicCard